tests/test_verifier.py

```diff
@@ -163,3 +166,106 @@ def test_get_file_info_local_good_input_wrong_size():
+            "size": 83761,
+            "uri": "http://opendata.cern.ch/eos/opendata/cms/software/HiggsExample20112012/HiggsDemoAnalyzer.cc",
+        },
+        {
+            "checksum": "adler32:46a907fc",
+            "name": "List_indexfile.txt",
+            "size": 1669,
+            "uri": "http://opendata.cern.ch/eos/opendata/cms/software/HiggsExample20112012/List_indexfile.txt",
+        },
+        {
+            "checksum": "adler32:af301992",
+            "name": "M4Lnormdatall.cc",
+            "size": 14943,
+            "uri": "http://opendata.cern.ch/eos/opendata/cms/software/HiggsExample20112012/M4Lnormdatall.cc",
+        },
+    ]
+
+    # Local test files info
+    test_file_info_local = [
+        {"name": "List_indexfile.txt", "size": 1669, "checksum": "adler32:46a907fc"},
+        {"name": "M4Lnormdatall.cc", "size": 14943, "checksum": "adler32:af301992"},
+        {"name": "BuildFile.xml", "size": 305, "checksum": "adler32:ff63668a"},
+        {"name": "HiggsDemoAnalyzer.cc", "size": 83761, "checksum": "adler32:f205f068"},
+    ]
+
+    # Removing the existing downloads
+    if os.path.isdir("5500/"):
+        shutil.rmtree("5500/")
+
+    # Simulating function call
+    assert verify_file_info(test_file_info_local, test_file_info_remote) is True
+
+    # Removing the test files directory
+    if os.path.isdir("5500/"):
+        shutil.rmtree("5500/")
+
+
+def test_verify_file_info_wrong_input():
+    """Test verify_file_info() for wrong input."""
+
+    # Remote test files info
+    test_file_info_remote = [
+        {
+            "checksum": "adler32:ff63668a",
+            "name": "BuildFile.xml",
+            "size": 305,
+            "uri": "http://opendata.cern.ch/eos/opendata/cms/software/HiggsExample20112012/BuildFile.xml",
+        },
+        {
+            "checksum": "adler32:f205f068",
+            "name": "HiggsDemoAnalyzer.cc",
+            "size": 83761,
+            "uri": "http://opendata.cern.ch/eos/opendata/cms/software/HiggsExample20112012/HiggsDemoAnalyzer.cc",
+        },
+        {
+            "checksum": "adler32:46a907fc",
+            "name": "List_indexfile.txt",
+            "size": 1669,
+            "uri": "http://opendata.cern.ch/eos/opendata/cms/software/HiggsExample20112012/List_indexfile.txt",
+        },
+        {
+            "checksum": "adler32:af301992",
+            "name": "M4Lnormdatall.cc",
+            "size": 14943,
+            "uri": "http://opendata.cern.ch/eos/opendata/cms/software/HiggsExample20112012/M4Lnormdatall.cc",
+        },
+    ]
+
+    # Local test files info
+    test_file_info_local = [
+        {"name": "List_indexfile.txt", "size": 1669, "checksum": "adler32:46a907fc"},
+        {"name": "M4Lnormdatall.cc", "size": 14943, "checksum": "adler32:af301992"},
+        {"name": "BuildFile.xml", "size": 305, "checksum": "adler32:ff63668a"},
+    ]
+
+    # Removing the existing downloads
+    if os.path.isdir("5500/"):
```

Comment:
Ditto here, if we are not downloading anything, do we need to pay attention to directories?

Reply:
done 👍